server/memcached/src/main/java/org/infinispan/server/memcached/binary/BinaryDecoder.java

```diff
@@ -26,19 +28,21 @@
  **/
 abstract class BinaryDecoder extends MemcachedBaseDecoder {
 
+   protected BinaryHeader singleHeader = new BinaryHeader();
```

Comment:
Unused?

Reply:
The generated decoder uses it. I added a method so it is more explicit.